### DISCUSSION TITLE
Background Tasks

### DISCUSSION THREAD

**Discussion by myokeeh** [microsoft/TemplateStudio#4588] [Q&A] [answered]
Is the WinUI3 version going to have Background Task or equivalent added to it? I currently have an OOP background task for a UWP app I'm converting.

Guidance on this page says I got to make a "Windows Runtime component" https://docs.microsoft.com/en-us/windows/apps/windows-app-sdk/migrate-to-windows-app-sdk/feature-mapping-table

Is that going to be added to TS?

**Comment by mikebattista** [comment]
It's not currently scheduled for any release. There are endless features we could add so it's a matter of priority and how much value it would add.

I'd be interested in hearing your migration experience to inform what any generated code might look like in a Template Studio project.

**Reply by myokeeh** [reply]
So far, I'm still held up by a blocker. Apparently, the sample code is out of date. I can't get the app to build as is per the sample.

**Reply by myokeeh** [reply]
https://github.com/microsoft/CsWinRT/discussions/ 1243

**Comment by myokeeh** [comment]
Is there better guidance on this topic?

**Reply by myokeeh** [reply, marked as answer]
https://github.com/microsoft/CsWinRT/issues/1327
